src/components/messenger/list/rewards-tooltip/container.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import { connectContainer } from '../../../../store/redux-container';
+import { RootState } from '../../../../store/reducer';
+import { rewardsTooltipClosed } from '../../../../store/rewards';
```

Comment:
I think we should rename this to `closeRewardsTooltip`. It makes more sense in the most contexts.